### ISSUE TITLE
Version 26 has issue with spawn_command_line_sync

### ISSUE THREAD

**Issue by ikelos** [bjarosze/gnome-bluetooth-quick-connect#50]
Hi there,

Every time I unlock my machine, I get a notification that Bluetooth Quick Connect (error trying to run 'bluetoothctl').  Digging around in the source showed that it was a problem with this line:
```
            GLib.spawn_command_line_sync("bluetoothctl --version");
```
which in LG yielded:
```
  r(0) = <exception TypeError: function GLib.spawn_command_line_sync: At least 2 arguments required, but only 1 passed>
```

Interestingly:

* A single parameter appears to be allowed in the latest documentation of the GLib code
* It doesn't seem to prevent the extension from working properly, it just throws up the issue at every unlock

```
GLib.spawn_command_line_sync.toString()
```
suggests it's expecting a command_line and an exit_status variable.

I'm using GLib version 2.70.2 assuming that's the library that the javascript bindings are based on.  It's all on Gentoo, so should all be compiled against the right versions, but happy to run tests if that will help...

**Comment by movq** [comment]
I had the same problem. It seems that the stable versions of `dev-libs/gobject-introspection` and `dev-libs/gobject-introspection-common` are too old (1.68) in Gentoo. I was able to fix it by unmasking 1.70.0 and updating these packages, then rebuilding `dev-libs/gjs`.

**Comment by ikelos** [comment]
Hmmm, I had 1.70.0, and rebuilding `gjs`/`glib` didn't seem to help, but rebuilding the `gobject-introspection` packages and then re-emerging gjs/glib seems to have worked.  Thanks!  5:D  I'll mark this as closed...  5:)
